src/__tests__/routes/__test__/protectedRoute.test.js

```diff
@@ -20,4 +22,23 @@ describe("protected routes", () => {
     };
     expect(Authenticate(token)).toEqual(response);
   });
+
+  it("should protectedRoute component should render without crashing", () => {
+    const wrapper = shallow(<ProtectedRoute />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("test valid token at login but expired", () => {
+    const falseToken = Authenticate(
+      "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6ImtpbWJzaW1vbjJAZ21haWwuY29tIiwiZXhwIjoxNTQyMDYxMjQ3LCJ1c2VybmFtZSI6ImtpbWJ1Z3AifQ.yz9jMoPBnKy9Tko_hhCwakNwZkeRLD4nc13A3LzyJbw",
```

Comment:
do you need this line here

Reply:
Yes i need to pass in a token that has expired so that it simulates how the component would react when a users token is no longer valid